pkg/sidecar/link.go

```diff
@@ -181,6 +182,29 @@ func (l *NetlinkLink) Shape(shape sync.LinkShape) error {
 	return nil
 }
 
+// TODO(cory) actually process the shape per network.
+// For now, this simply adds a route based on the Filter
+func (l *NetlinkLink) AddRules(rules []network.LinkRule) error {
+	for _, rule := range rules {
+		r := netlink.Route{
+			Dst: &rule.Subnet,
+		}
+		switch rule.Filter {
+		case network.Accept:
+			continue
```

Comment:
Will this work for the case where we're going _from_ blocking _to_ accepting a subnet?

Reply:
that's a good point, you should be able to accept later once you've blocked it.